micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/correlationid/CorrelationIdFilter.groovy

```diff
@@ -61,16 +80,21 @@ class CorrelationIdFilter extends OncePerRequestFilter {
         return correlationId
     }
 
-    //TODO: add microservice id to corrId, so that we know where it was created
-    private String createNewCorrIdIfEmpty(String currentCorrId) {
-        if (!hasText(currentCorrId)) {
-            currentCorrId = UUID.randomUUID().toString()
-            MDC.put(CORRELATION_ID_HEADER, currentCorrId)
-            log.debug("Generating new correlationId: " + currentCorrId)
-        }
+    private String createNewCorrIdIfEmpty() {
```

Comment:
I see.

Btw, probably it is not in a scope of that issue, but I realized that we have correlation ID generation in 6 different places as `UUID.randomUUID().toString()`. Maybe to keep it consistent we could have `CorrelationIdGenerator`?